app/assets/js/rtMedia.backbone.js

```diff
@@ -413,30 +413,26 @@ jQuery( function( $ ) {
 			e.preventDefault();
 
 			if ( '' === $( '#media_search_input' ).val() ) {
```

Comment:
Can we please cache `$( '#media_search_input' ).val()`
`var $media_search_input=$( '#media_search_input' ).val();`
so we can just use cached `$media_search_input` further in code and not make jquery calls twice for same thing.